src/framework/global/internal/interactive.cpp

```diff
@@ -239,8 +239,9 @@ Ret Interactive::revealInFileBrowser(const io::path& filePath) const
         return true;
     }
 #elif defined(Q_OS_WIN)
-    QString command = QLatin1String("explorer /select,%1").arg(QDir::toNativeSeparators(filePath.toQString()));
-    if (QProcess::startDetached(command)) {
+    QString program = QLatin1String("explorer");
+    QStringList arguments = QStringList(QString("/select,%1").arg(QDir::toNativeSeparators(filePath.toQString())));
```

Comment:
Are you sure that this still works with file names that contain spaces? On the internet, I read that Qt will add quotation marks around arguments that contain spaces, which is does not work in this case. 
Probably, the solution is to keep it as it was, and only add `QStringList` as a parameter so that we are using the non-obsolete overload.

Reply:
I haven't tried it. And don't understand what you mean to be the solution?

Reply:
But to deal with spaces maybe use
`QStringList arguments = QStringList(QString("/select,\"%1\"").arg(QDir::toNativeSeparators(filePath.toQString())));`

Reply:
Yes, making it `if (QProcess::startDetached(command, QStringList())) {` does seem to work(around that "obsolete" warning).

Reply:
You like that better?

Reply:
> `if (QProcess::startDetached(command, QStringList())) {`

Yes, I think that is the best option.

Reply:
Done